lab1/.idea/lab1.iml

```diff
@@ -2,7 +2,10 @@
 <module type="PYTHON_MODULE" version="4">
```

Comment:
do you really need this file? (or the `.idea` folder for that matter)

Reply:
Don't think so cause this is just a lab, but in case of 2+ dev you might want to share some configs.